Add User Feedback alerts to main Alerts docs page (#11446)
getsentry/sentry-docs: docs/product/alerts/alert-types.mdx

### Alert Details Page

The **Alert Details** page shows a list of uptime issues that were created by the alert. Clicking on any of the issues will take you to a page with additional information that may help you debug that issue.

## User Feedback Alerts

You can enable alerts for the [User Feedback Widget](/product/user-feedback/#user-feedback-widget) on the [issue alerts](https://sentry.io/orgredirect/organizations/:orgslug/alerts/new/issue/) page by following the steps below:

1. Create a [New Alert Rule](https://sentry.io/orgredirect/organizations/:orgslug/alerts/new/issue/) in Sentry.
2. Scroll to the "Set conditions" section and set the "IF" filter to `The issue's category is equal to… "Feedback"`.
3. Choose which actions to perform in the “THEN” filter.
4. Add an alert name and owner.

> ![User Feedback Alert Configuration.](./img/user-feedback-alert-configuration.png)

Author: ```suggestion
![User Feedback Alert Configuration.](../user-feedback/img/user-feedback-alert-configuration.png)
```

> User Feedback alerts for the [User Feedback Widget](/product/user-feedback/#user-feedback-widget) can be enabled under issue alerts. Follow these steps to set up alerts for every new feedback:
> 
> 1. Create a [New Alert Rule](https://sentry.io/alerts/new/issue/) in Sentry.
> 2. Scroll to the "Set conditions" section and set the "IF" filter to `The issue's category is equal to… "Feedback"`.
> 3. Choose which actions to perform in the “THEN” filter.
> 4. Add an alert name and owner.
> 
> ![User Feedback Alert Configuration.](./img/user-feedback-alert-configuration.png)
> 
> To get alerts when [crash-report feedback](/product/user-feedback/#crash-report-modal) comes in, make sure to turn on "Enable Crash Report Notifications" in Settings > Projects > [Project Name] > User Feedback.

Reviewer: ```suggestion
You can enable alerts for the [User Feedback Widget](/product/user-feedback/#user-feedback-widget) on the [issue alerts](https://sentry.io/orgredirect/organizations/:orgslug/alerts/new/issue/) page by following the steps below:

1. Create a [New Alert Rule](https://sentry.io/orgredirect/organizations/:orgslug/alerts/new/issue/) in Sentry.
2. Scroll to the "Set conditions" section and set the "IF" filter to `The issue's category is equal to… "Feedback"`.
3. Choose which actions to perform in the “THEN” filter.
4. Add an alert name and owner.

![User Feedback Alert Configuration.](./img/user-feedback-alert-configuration.png)

To get alerts when [crash-report feedback](/product/user-feedback/#crash-report-modal) comes in, make sure to turn on "Enable Crash Report Notifications" in Settings > Projects > [Project Name] > User Feedback.
```

Reviewer: @jas-kas Looks like the image you're trying to add isn't showing up :(

Reviewer 2: ooh @jas-kas you might need to do `../user-feedback/img/user-feedback-alert-configuration.png` since that img isn't in the same folder as this one
